N8N integration (#21835)
DataDog/integrations-core: n8n/README.md

# Agent Check: n8n

## Overview

This check monitors [n8n][1] through the Datadog Agent. 

Include a high level overview of what this integration does:
- What does your product do (in 1-2 sentences)?
- What value will customers get from this integration, and why is it valuable to them?

> - What specific data will your integration monitor, and what's the value of that data?

Reviewer: Please add a meaningful description, or remove these lines.

# Agent Check: n8n

## Overview

This check monitors [n8n][1] through the Datadog Agent. 

Collect n8n metrics including:
- Cache metrics: Hit and miss statistics.
- Message event bus metrics: Event-related metrics.
- Workflow metrics: Can include workflow ID labels.
- Node metrics: Can include node type labels.
- Credential metrics: Can include credential type labels.
- Queue Metrics

## Setup

Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.

### Installation

The n8n check is included in the [Datadog Agent][2] package.
No additional installation is needed on your server.

### Configuration

The `/metrics` endpoint is disabled by default and must be enabled by setting `N8N_METRICS`=`true`. You can also customize the metric prefix using `N8N_METRICS_PREFIX` (default is `n8n_`).

Note that the `/metrics` endpoint is only available for self-hosted instances and is not available on n8n Cloud.

> For the datadog agent to collect metrics, you will need to follow the instructions provided [here][10].

Reviewer: ```suggestion
For the Datadog Agent to collect metrics, follow the instructions to [enable Prometheus metrics][10].
```
